microsoft-365/admin/setup/customize-your-organization-theme.md

```diff
@@ -18,14 +18,14 @@ search.appverid:
 - MET150
 - MOE150
 ms.assetid: 8275da91-7a48-4591-94ab-3123a3f79530
-description: "Learn to change the default theme of Office 365 and customize it to match your company logo or color. "
+description: "Learn to change the default theme of Microsoft 365 and customize it to match your company logo or color. "
 ---
 
-# Customize the Office 365 theme for your organization
+# Customize the Microsoft 365 theme for your organization
 
-Learn how to customize your theme in the Microsoft 365 admin center. As the admin of your Office 365 for business subscription, you can change the default theme that appears in the top navigation bar for everyone in the organization. You can add your company logo and change the colors to match the rest of your brand. You can even add a destination link for users to navigate to when they select your logo. You can see here the default theme and custom theme result in Office 365.
+Learn how to customize your theme in the Microsoft 365 admin center. As the admin of your Microsoft 365 for business subscription, you can change the default theme that appears in the top navigation bar for everyone in the organization. You can add your company logo and change the colors to match the rest of your brand. You can even add a destination link for users to navigate to when they select your logo. You can see here the default theme and custom theme result in Microsoft 365.
```

Comment:
```suggestion
Learn how to customize your theme in the Microsoft 365 admin center. As the admin of your Microsoft 365 Business subscription, you can change the default theme that appears in the top navigation bar for everyone in the organization. You can add your company logo and change the colors to match the rest of your brand. You can even add a destination link for users to navigate to when they select your logo. You can see here the default theme and custom theme result in Microsoft 365.
```

Reply:
Per Project Snowbird (rebranding Office 365 to Microsoft 365), all instances of 'Office 365 business' change to 'Microsoft 365 Apps for business.'